tools/gulp/util/firebase.ts

```diff
@@ -6,18 +6,26 @@ const cloudStorage = require('@google-cloud/storage');
 const screenshotFirebaseConfig = require('../../screenshot-test/functions/config.json');
 
 /** Opens a connection to the Firebase dashboard app. */
-export function openFirebaseDashboardApp() {
+export function openFirebaseDashboardApp(asGuest = false) {
+  const databaseURL = 'https://material2-board.firebaseio.com';
```

Comment:
I only did it like that because the `firebase-admin` also uses it like that and so I can just pass it like that. But can change it.